src/main/java/com/google/codeu/data/UserDatastore.java

```diff
@@ -99,5 +100,19 @@ public User getUser(String id) {
 	    users.add((String) entity.getProperty("email"));
 	  }
 	  return users;
-	}
+  }
+  
+  public Set<String> getUsersForTeam(UUID teamId) {
```

Comment:
We should return a list of User models(the UI will only show the names of the users but it will need other data like the id to render links to those users profiles. 

Also the parameter can be a String.

```suggestion
  public List<User> getUsersForTeam(String teamId) {
```